src/Middleware/RequestThrottling/perf/Microbenchmarks/QueueFullOverhead.cs

```diff
@@ -24,16 +23,10 @@ public class QueueFullOverhead
         [GlobalSetup]
         public void GlobalSetup()
         {
-            var options = new RequestThrottlingOptions
-            {
-                MaxConcurrentRequests = MaxConcurrentRequests,
-                RequestQueueLimit = _numRequests
-            };
-
-            _middleware = new RequestThrottlingMiddleware(
-                    next: (RequestDelegate)_incrementAndCheck,
-                    loggerFactory: NullLoggerFactory.Instance,
-                    options: Options.Create(options)
+            _middleware = TestUtils.CreateTestMiddleware_TailDrop(
+                maxConcurrentRequests: MaxConcurrentRequests,
+                requestQueueLimit: _numRequests,
+                next: (RequestDelegate)_incrementAndCheck
```

Comment:
I just saw that `_incrementAndCheck` is a method not a field. Make it `IncrementAndCheck`.

If you were going to pass around an IncrementAndCheck RequestDelegate a lot and it captured `this` it would make sense to store it in a field in the constructor to avoid allocating each time. But since you only pass it as an argument once during global setup, the implicit closure is fine.